### PR TITLE
:boom: remove unnecessary emr instanceCount config (orchard 0.15.4)

### PR DESCRIPTION
**Pull request by realstraw**

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
- ThisBuild / version := "0.11.1"
+ ThisBuild / version := "0.13.0"
```
Comment (schowsf) [inline version.sbt]: is skipping 0.12.0 deliberate?  @realstraw

**Comment by damianxu88** [comment]
@realstraw unintentonally closed this PR, reopened now

File: spade-aws/src/main/scala/com/salesforce/mce/spade/aws/resource/EmrCluster.scala [hunk 2]
```diff
@@ -115,7 +115,6 @@ object EmrCluster {
           configurations.map(_.asSpec()).asOption(),
           EmrResourceSpec.InstancesConfig(
             subnetId.getOrElse(sac.emr.subnetId),
-            instanceCount,
```
Comment (anniezhang77) [inline spade-aws/src/main/scala/com/salesforce/mce/spade/aws/resource/EmrCluster.scala:118]: this is no longer needed?

File: version.sbt (hunk 1 above)
Comment (realstraw) [reply version.sbt]: oh, that was a mistake. apparently, I can't count

File: spade-aws/src/main/scala/com/salesforce/mce/spade/aws/resource/EmrCluster.scala (hunk 2 above)
Comment (realstraw) [reply spade-aws/src/main/scala/com/salesforce/mce/spade/aws/resource/EmrCluster.scala:118]: yeah, it doesn't do anything

**Review by schowsf — APPROVED** [review]

**Review by damianxu88 — APPROVED** [review]

**Pull request merged**